packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -209,6 +209,62 @@ const conf = (module.exports = convict({
         format: 'int',
       },
     },
+    betaReact: {
+      simpleRoutes: {
+        default: false,
+        doc: 'Enable users visiting the simple routes behind the beta url',
+        format: Boolean,
+        env: 'REACT_CONVERSION_SIMPLE_ROUTES',
+      },
+      resetPassword: {
+        default: false,
+        doc: 'Enable users visiting the routes associated with reset_password behind the beta url',
+        format: Boolean,
+        env: 'REACT_CONVERSION_RESET_PASSWORD',
+      },
+      oauth: {
+        default: false,
+        doc: 'Enable users visiting the oauth routes behind the beta url',
+        format: Boolean,
+        env: 'REACT_CONVERSION_OAUTH',
+      },
+      signIn: {
+        default: false,
+        doc: 'Enable users visiting the signin routes behind the beta url',
+        format: Boolean,
+        env: 'REACT_CONVERSION_SIGNIN',
+      },
+      signUp: {
+        default: false,
+        doc: 'Enable users visiting the signup routes behind the beta url',
+        format: Boolean,
+        env: 'REACT_CONVERSION_SIGNUP',
+      },
+      pair: {
+        default: false,
+        doc: 'Enable users visiting the pair routes behind the beta url',
+        format: Boolean,
+        env: 'REACT_CONVERSION_PAIR',
+      },
+      postVerifyAddRecoveryKey: {
+        default: false,
+        doc: 'Enable users visiting the post-verify add recovery key routes behind the beta url',
+        format: Boolean,
+        env: 'REACT_CONVERSION_POST_VERIFY_ADD_RECOVERY_KEY',
+      },
+      postVerifyCADviaQR: {
```

Comment:
Nit: Not sure if it matters for anything, but the `via` here seems to be inconsistent w/ the `Via` below on L261.
But not sure how to "fix" that apart from renaming to `postVerifyCadViaQr`. 🤷